_benchmark/quickstart.md

```diff
@@ -0,0 +1,421 @@
+
+After the installation completes, verify that OSB is running by entering the following command:
+
+```bash
+opensearch-benchmark --help
+```
+{% include copy.html %}
+
+If successful, OpenSearch returns the following response:
+
+```bash
+$ opensearch-benchmark % opensearch-benchmark --help
+usage: opensearch-benchmark [-h] [--version] {execute-test,list,info,create-workload,generate,compare,download,install,start,stop} ...
+
+   ____                  _____                      __       ____                  __                         __
+  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
+ / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
+/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
+\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
+    /_/
+
+ A benchmarking tool for OpenSearch
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --version             show program's version number and exit
+
+subcommands:
+  {execute-test,list,info,create-workload,generate,compare,download,install,start,stop}
+    execute-test        Run a benchmark
+    list                List configuration options
+    info                Show info about a workload
+    create-workload     Create a Benchmark workload from existing data
+    generate            Generate artifacts
+    compare             Compare two test_executions
+    download            Downloads an artifact
+    install             Installs an OpenSearch node locally
+    start               Starts an OpenSearch node locally
+    stop                Stops an OpenSearch node locally
+
+Find out more about Benchmark at https://opensearch.org/docs
+```
+
+### Docker
+
+To pull the image from Docker Hub, run the following command:
+
+```bash
+docker pull opensearchproject/opensearch-benchmark:latest
+```
+{% include copy.html %}
+
+Then, run the Docker image:
+
+```bash
+docker run opensearchproject/opensearch-benchmark -h
+```
+{% include copy.html %}
+
+
+## Running your first Benchmark
+
+You can now run your first benchmark. For your first benchmark, you'll use the [geonames](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/geonames) workload.
+
+
+### Understanding workload command flags
+
+Benchmarks are run using the [`execute-test`]({{site.url}}{{site.baseurl}}/benchmark/commands/execute-test/) with the following command flags:
+
+For additional `execute_test` command flags, see the [execute-test]({{site.url}}{{site.baseurl}}/benchmark/commands/execute-test/) reference. Some commonly used options are `--workload-params`, `--exclude-tasks`, and `--include-tasks`
+{: .tip}
+
+* `--pipeline=benchmark-only` : Informs OSB that you wants to provide their own OpenSearch cluster.
```

Comment:
`Informs OSB that you wants to provide their own OpenSearch cluster.` --> `Informs OSB that users wants to provide their own OpenSearch cluster.`
their to users